CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### libcontainer API
+ * `configs.CommandHook` struct has changed, Command is now a pointer.
+   Also, `configs.NewCommandHook` now accepts a `*Command`. (#4325)
```

Comment:
Could you explain why we need to change this struct? Is it necessary?

Reply:
In fact, I think this PR should not be blocked by these small things, we should change as little as possible about the logic of hooks.

Reply:
Ah, it's about type assertion and the member var modification, make sense.

Reply:
Yes, otherwise it's not possible to modify the struct member (`Env`) in place, and the way the whole thing is structured, we only have final environment in a very late stage.

The alternatives are to either pass `env` to `RunHooks` as an additional argument, which I tried and it looks much worse.